test/v1/configuration.go

```diff
@@ -38,6 +38,10 @@ import (
 // that uses the image specified by names.Image.
 func CreateConfiguration(t pkgTest.T, clients *test.Clients, names test.ResourceNames, fopt ...rtesting.ConfigOption) (*v1.Configuration, error) {
 	config := Configuration(names, fopt...)
+	if config.Labels == nil {
+		config.Labels = map[string]string{}
+	}
+	config.Labels[test.TestLabel] = "true"
```

Comment:
Do we just `true` or (`1`) or perhaps more involved like date/time?

Reply:
Datetime isn't that useful because the resource already has a created timestamp. I am going to try switching it to test name which might be more interesting for debugging.